src/hid-sensor.cpp

```diff
@@ -221,6 +221,9 @@ void hid_sensor::start( frame_callback_ptr callback )
                                         << rs2_timestamp_domain_to_string( timestamp_domain ) << ",last_frame_number,"
                                         << last_frame_number << ",last_timestamp," << last_timestamp );
 
+            if( frame_counter <= last_frame_number )
+                LOG_INFO( "Frame counter reset" );
```

Comment:
I suggest LOG_DEBUG.

Since this is expected behavior, we should not list it as information to the user (viewer log for example)

Reply:
This is what it was before... and by default user doesn't even see INFO.
But OK, if that's what you want...